codespell_lib/data/dictionary.txt

```diff
@@ -21822,6 +21840,7 @@ Mocrosoft->Microsoft
 mocule->module
 mocules->modules
 moddel->model
+moddeled->modeled
```

Comment:
Shall we also include `en-GB` variant?
```text
moddelled->modelled
```
* https://github.com/search?q=moddeled&type=code  231
* https://github.com/search?q=moddelled&type=code  47